lib/loadtest.js

```diff
@@ -297,15 +297,15 @@ function HttpClient(operation, params)
 			});
 			connection.on('end', function()
 			{
-				if (connection.statusCode >= 400)
-				{
-					return callback('Status code ' + connection.statusCode, connection.statusCode);
-				}
 				var result = {
 					statusCode: connection.statusCode,
 					body: body,
 					headers: connection.headers,
 				};
+				if (connection.statusCode >= 400)
+				{
+					return callback(result, connection.statusCode);
```

Comment:
For consistency, please return the result in the same order, and do not change the first parameter:

    return callback('Status code ' + connection.statusCode, result);